daemon/algod/server.go

```diff
@@ -227,30 +225,28 @@ func (s *Server) Start() {
 	c := make(chan os.Signal)
 	signal.Notify(c, os.Interrupt, syscall.SIGTERM, syscall.SIGINT)
 	signal.Ignore(syscall.SIGHUP)
-	go func() {
-		sig := <-c
+
+	fmt.Printf("Node running and accepting RPC requests over HTTP on port %v. Press Ctrl-C to exit\n", addr)
+	select {
+	case err := <-errChan:
+		if err != nil {
```

Comment:
errChan name is giving the impression that only errors will trigger this case.
However, this case is triggered when the server returns.
Maybe rename errChan to serverReturn?

Reply:
I will look into that in a subsequent PR.